lambdas/src/apis/profiles/controllers/profiles.ts

```diff
@@ -1,35 +1,28 @@
 import { Request, Response } from 'express'
-import fetch from "node-fetch"
-import { Environment, EnvironmentConfig, Bean } from '../../../Environment'
 import { filterENS } from '../ensFiltering'
 import { SmartContentServerFetcher } from '../../../SmartContentServerFetcher'
-import { Fetcher } from 'dcl-catalyst-commons'
 
-export async function getProfileById(env: Environment, smartContentServerFetcher: SmartContentServerFetcher, req: Request, res: Response) {
+export async function getProfileById(fetcher: SmartContentServerFetcher, ensOwnerProviderUrl: string, req: Request, res: Response) {
     // Method: GET
     // Path: /:id
     const profileId:string = req.params.id
-    const v3Url = (await smartContentServerFetcher.getContentServerUrl()) + `/entities/profile?pointer=${profileId}`
-    const response = await fetch(v3Url)
     let returnProfile: EntityMetadata = { avatars:[] }
-    if (response.ok) {
-        const entities:V3ControllerEntity[] = await response.json()
+    try {
+        const entities:V3ControllerEntity[] = await fetcher.fetchJsonFromContentServer(`/entities/profile?pointer=${profileId}`)
```

Comment:
Can we start using the `/deployments` endpoint?
It would be something like `/deployments?entityType=profile&pointer=${profileId}&onlyCurrentlyPointed=true?fields=metadata`

In a next iteration, it would also be nice to start using the [client](https://github.com/decentraland/catalyst-client)

Reply:
I love the /deployments endpoint for obvious reasons however, I think this endpoint is a lot clearer because is entity oriented.
Agree with you regarding the client, but I would prefer to make that change in another iteration.